repository_service_tuf/cli/admin/helpers.py

```diff
@@ -405,17 +377,24 @@ def _add_root_signatures_prompt(
 
         results = _filter_root_verification_results(root_result)
         keys = _print_keys_for_signing(results)
-
-        choice = _choose_signing_key_prompt(
-            len(keys), allow_skip=bool(root_result.signed)
-        )
-        if choice == -1:
+        key_choices = {
+            key.unrecognized_fields.get(KEY_NAME_FIELD, key.keyid): key
+            for key in keys
+        }
+        sign_options = list(key_choices.keys())
+        or_continue = ":"
+        if bool(root_result.signed):
+            or_continue = " or continue:"
+            sign_options.insert(0, "continue")
+
+        console.print(f"\nSelect one current key to sign{or_continue}")
```

Comment:
Seems like this sentence could be better formulated.
What about:
`f"\nSelect a key for signing{or_continue}"`